definitions/artifacts/aws-dynamodb-table.json

```diff
@@ -1,8 +1,8 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema",
-  "$id": "https://api.massdriver.cloud/artifact-definitions/massdriver/aws-dynamodb-table.json",
   "$md": {
-    "access": "public"
+    "access": "public",
+    "name": "aws-dynamodb-table"
```

Comment:
If we don't scope this to org as `massdriver/aws-dynamodb-table`, then I won't be able use this as the `type` field in the artifact.

In the artifact specification (https://github.com/massdriver-cloud/aws-vpc/blob/main/massdriver.yaml#L58) we use the `massdriver/<artifact>` format which is what I'm sending to you.

We should consolidate around a format - either have it both places or not have it in either place, right?

Reply:
This will be set by the backend in the $id when published